code/modules/client/verbs/ooc.dm

```diff
@@ -28,11 +28,11 @@
 			usr << "<span class='danger'>OOC for dead mobs has been turned off.</span>"
 			return
 		if(prefs.muted & MUTE_OOC)
```

Comment:
Unnecessary I think. This is not for ban, it's for OOC turned off by player in options.